src/DebugSceneList.js

```diff
@@ -23,6 +23,7 @@ const styles = StyleSheet.create({
     alignItems: 'center',
   },
   componentWrapper: {
+    marginHorizontal: 20,
```

Comment:
I thought we didn't want this and instead set the margin on Individual components if needed.